contracts/facade/FacadeWrite.sol

```diff
@@ -33,12 +33,15 @@ contract FacadeWrite is IFacadeWrite {
             require(setup.backups[i].backupCollateral.length > 0, "no backup collateral");
         }
 
-        // Validate beneficiary was set intentionally
-        require(
-            setup.beneficiary != address(0) ||
-                (setup.revShare.rTokenDist == 0 && setup.revShare.rsrDist == 0),
-            "beneficiary revShare mismatch"
-        );
+        // Validate beneficiaries
+        for (uint256 i = 0; i < setup.beneficiaries.length; ++i) {
+            require(
+                setup.beneficiaries[i].beneficiary != address(0) ||
+                    (setup.beneficiaries[i].revShare.rTokenDist == 0 &&
+                        setup.beneficiaries[i].revShare.rsrDist == 0),
```

Comment:
These requires that were here before were because you _had_ to supply a beneficiary. In the world where there are multiple beneficiaries and the array can just be zero length, then I think the require should be:
`require(setup.beneficiaries[i].beneficiary != address(0) && (setup.beneficiaries[i].revShare.rTokenDist > 0 || setup.beneficiaries[i].revShare.rsrDist > 0)`

Reply:
ok, makes sense. there is no need for a 0 distribution in the original setup

Reply:
Done